content/docs/command-reference/remote/modify.md

```diff
@@ -75,6 +75,22 @@ The following config options are available for all remote types:
   $ dvc remote modify localremote url /home/user/dvcstore
   ```
 
+- `jobs` - number of workers started at execution of the following commands:
+
+  - push
+  - pull
+  - status
+  - gc
+  - fetch
+
```

Comment:
Thanks for the clarification (per https://github.com/iterative/dvc.org/pull/1941#pullrequestreview-530560968)!

I would simplify this a little, to:
```suggestion
- `jobs` - number of processes to perform [remote storage](link)
  synchronization commands (`push`, `pull`, `fetch` `status -c`, and
  `gc -c`).
```
> Needs formatting (esp. once merged with the paragraph below).